Source/CBLRestPusher.m

```diff
@@ -603,7 +606,10 @@ - (void) uploadJSONRevision: (CBL_Revision*)originalRev {
                       body: rev.properties
               onCompletion: ^(id response, NSError *error) {
                   if (error) {
-                      self.error = error;
+                      // 403/Forbidden means validation failed; don't treat it as an error
+                      // because I did my job in sending the revision.
+                      if (error.code != kCBLStatusForbidden)
```

Comment:
Same comment as above. Since this logic is in 3 places now, maybe it should be factored out into a method?